manage.py

```diff
@@ -140,6 +140,48 @@ def _add_live_test_subparser(subparsers):
     )
 
 
+def _add_validate_schema_subparser(subparsers):
+    schema_validation_usage = 'manage.py validate-schemas [options]'
+    schema_validation_description = ("""
+StreamAlertCLI v{}
+Run end-to-end tests that will attempt to send alerts
+
+Available Options:
+
+    --files                 Name (not full path) of test file(s) to validate, separated by spaces
```

Comment:
Should we call this `--test-files`? I'd also add a little description of what these files should contain